launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -208,3 +172,53 @@ def execute(
                 None, self._load_in_sequence, self.__composable_node_descriptions, context
             )
         )
+
+
+def get_composable_node_load_request(
+    composable_node_description: ComposableNode,
+    context: LaunchContext
+):
+    """Get the request that will be send to the composable node container."""
+    request = composition_interfaces.srv.LoadNode.Request()
+    request.package_name = perform_substitutions(
+        context, composable_node_description.package
+    )
+    request.plugin_name = perform_substitutions(
+        context, composable_node_description.node_plugin
+    )
+    if composable_node_description.node_name is not None:
+        request.node_name = perform_substitutions(
+            context, composable_node_description.node_name
+        )
+    if composable_node_description.node_namespace is not None:
+        request.node_namespace = perform_substitutions(
+            context, composable_node_description.node_namespace
+        )
+    # request.log_level = perform_substitutions(context, node_description.log_level)
+    if composable_node_description.remappings is not None:
+        for from_, to in composable_node_description.remappings:
+            request.remap_rules.append('{}:={}'.format(
+                perform_substitutions(context, list(from_)),
+                perform_substitutions(context, list(to)),
+            ))
+    global_params = context.launch_configurations.get('ros_params', None)
+    parameters = composable_node_description.parameters
+    if global_params is not None:
+        parameters = (normalize_parameter_dict(global_params), *parameters)
+    if composable_node_description.parameters is not None:
```

Comment:
@ivanpauno this should occur if only global parameters are available too.

Reply:
fixed https://github.com/ros2/launch_ros/pull/158/commits/f66375545bf968f686c60e94fbc961f853b5109c